src/components/StartPage.vue

```diff
@@ -1,172 +1,70 @@
 <template>
-  <q-icon
-    :name="$q.dark.isActive ? 'fas fa-sun' : 'fas fa-moon'"
-    :class="[
-      $q.dark.isActive ? 'text-white' : 'text-black',
-      'cursor-pointer fixed-top-right q-mx-xl q-my-lg text-h5',
-    ]"
-    @click="toggleDarkMode"
-  />
-  <div
-    class="q-mx-auto q-my-xl"
-    style="max-width: 600px"
-  >
-    <h5 class="text-h4 q-mb-sm">
-      NER Text Annotator
-    </h5>
-    <p :class="['text-subtitle1', $q.dark.isActive ? 'text-grey-4' : 'text-grey-7']">
+  <div class="q-mx-auto q-my-xl" style="max-width: 600px">
+    <h1 class="text-h4 q-mb-sm text-center">NER Text Annotator</h1>
+    <p class="text-subtitle1 text-center">
       Annotate text for spaCy NER Model training
     </p>
 
     <div class="q-my-xl q-py-md column">
-      <div class="row justify-between items-center">
-        <div>
-          <p :class="['text-h6 q-my-md', $q.dark.isActive ? 'text-grey-4' : 'text-grey-7']">
-            Select your file(s), then press 'Begin'!
-          </p>
-          <p :class="['text-subtitle1 q-my-md', $q.dark.isActive ? 'text-grey-4' : 'text-grey-7']">
-            Hint: You can also drag and drop files into this window!
+      <div class="row justify-around q-my-xl">
+        <div class="col-5 q-mx-sm">
+          <q-file
+            v-model="textFile"
+            accept=".txt"
+            filled
+            label="Load a text file"
+            :color="highlightTextFileInput ? 'red-8' : 'primary'"
+            :bg-color="highlightTextFileInput ? 'red-1' : 'grey-3'"
+            @rejected="
+              fileSelectionError(
+                'Only text files (.txt) can be used for creating annotations.'
+              )
+            "
+          >
+            <template v-slot:prepend>
+              <q-icon name="fas fa-file-text" />
+            </template>
+          </q-file>
+
+          <p class="text-caption q-mx-sm q-my-sm text-accent">
+            You can also drag and drop a file into this window!
           </p>
         </div>
+        <div class="col-5 q-mx-sm">
+          <q-file
+            v-model="annotationFile"
+            accept=".json"
+            filled
+            label="Load annotations"
+            color="teal"
+            @rejected="fileSelectionError('Invalid annotation file')"
+          >
+            <template #prepend>
+              <q-icon name="fas fa-file-code" />
+            </template>
+          </q-file>
+          <p class="text-caption q-mx-sm q-my-sm text-accent">Optional</p>
```

Comment:
Opinion: I personally liked `text-grey-8` better instead of `text-accent`. `text-accent` looks purplish to me which feels like it clashes with the rest of the color scheme. Same for the 'You can also drag and drop a file into this window!' text. What do you think?